### ISSUE TITLE
Floater remover 

### ISSUE THREAD

**Issue by alexey7783** [playcanvas/supersplat#252]
A function that recognizes and selects floater, so they can be removed. For example with two values for "number of floats in a cluster" and "distance between clusters". Now the algorithm goes through the list of splats and looks for chunks of splats that float alone in space.

**Comment by lesath3d** [comment]
yes! selecting by distance between splats would be super useful!
